test/test-component.js

```diff
@@ -200,6 +200,19 @@ describe('Nested Component instance', function() {
       })
     })
 
+    it('successfully finds the panel root', function(done) {
```

Comment:
It "successfully finds a panel parent node by a given tag name." We could make a separate `describe('findPanelParentByTagName')` block here, but maybe that's overkill at this stage.